10.9-libcxx/stable/main/finkinfo/libs/libnewt0-shlibs.info

```diff
@@ -1,6 +1,6 @@
 Package: libnewt0-shlibs
-Version: 0.52.19
-Revision: 1.3
+Version: 0.52.20
+Revision: 1
```

Comment:
make this 5.1 please

Reply:
@dhomeier ping on @TheSin- 's request to change the Revision.